examples/mockups/product.html

```diff
@@ -0,0 +1,466 @@
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Automated Tests</div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">About the engineering workflow</a>
+            </div>
+          </div>
+
+          <div class="column">
+            <div>
+              <h3 class="ui header"><<i>missing</i>> Illustration</h3>
+              <h3 class="ui header">Write with reStructuredText or Markdown</h3>
+              <p>
+                Write documentation alongside your code, with all your same tools, enabling a culture where writers and developers both feel ownership of documentation and work together to make it as good as possible.
+              </p>
+              
+              <div class="ui relaxed divided list">
+
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Documentation  culture empowerment</div>
+                  </div>
+                </div>
+
+              </div>
+
+            </div>
+            <div>
+              <a href="" class="ui bui button basic inverted fluid large">Using RTD as a technical writer</a>
+            </div>
+          </div>
+
+          <div class="column">
+            <div>
+              <h3 class="ui header"><<i>missing</i>> Illustration</h3>
+              <h3 class="ui header">Easy Documentation Deployment for Technical Teams</h3>
+              <p>
+                We deploy and host your team's documentation automatically. With Pull Request previews you can validate all your changes, before they go live. Once you commit, we keep your documentation up to date, across all versions of your software.
+              </p>
+              
+              <div class="ui relaxed divided list">
+
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Always up to date</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Automatic deploy</div>
+                  </div>
+                </div>
+                
+                <div class="item">
+                  <i class="icon check circle green"></i>
+                  <div class="content">
+                    <div class="description">Multiple Versions</div>
+                  </div>
+                </div>
```

Comment:
Another piece we will likely want in this page is a the ability to call out to a feature of RTD for business. We don't have a succinct adjective for this currently, "premium" is not something we've used yet, but for example:

![image](https://user-images.githubusercontent.com/1140183/144315910-bec86044-b03c-46f8-8860-32cd0edde3f5.png)

Maybe label isn't the best element there, due to the line height on the list. A link might be enough, or something like a popup? Anyways, something to maybe think about short term and long term. It's probably something we'll end up wanting to include on the community site. We'll likely also want to call out to "Gold" features -- like disabling ads.

Reply:
I really like that basic label used as a badge for "premium"! We can explore other options but I like the effect.
Probably we'll want that badge to have the primary color we'll settle on for the corporate site. For now I'm assuming @orange.

Reply:
Or perhaps even calling out to the plan that the feature starts on? `Advanced plan` or `Advanced` In this case color coded to the pricing page plans even.

For ultimate fanciness, a plan color coded label that expands out to the plan name: `Advanced plan`